src/lib/inc/util.h

```diff
@@ -79,7 +79,7 @@ static inline bool range_in_range(unsigned long base1, unsigned long size1, unsi
 #define DEFINED(MACRO)                  _DEFINED(MACRO)
 #define _DEFINED_1                      0,
 #define _DEFINED(VALUE)                 __DEFINED(_DEFINED_##VALUE)
-#define __DEFINED(VALUE)                ___DEFINED(VALUE true, false)
+#define __DEFINED(VALUE)                ___DEFINED(VALUE true, false, dummy)
```

Comment:
@DavidMCerdeira Yes. ISO C does not allow empty ellipses arguments which happens in one of the cases of this macro chain. But it would be a good idea to clarify the original error in the commit message.

Reply:
Yes I agree

Reply:
Fixed in commit 6c69e52d29556667ce66240bbe0306974edf05cb

Reply:
Fixed in commit 6c69e52d29556667ce66240bbe0306974edf05cb